tests/foreman/api/test_contentviewversion.py

```diff
@@ -711,9 +711,12 @@ def test_positive_delete_cv_promoted_to_multi_env(module_org):
         content_view.read()
 
 
-@pytest.mark.stubbed
+@pytest.mark.no_containers
+@pytest.mark.rhel_ver_list([7, 8, 9])
 @pytest.mark.tier3
-def test_positive_remove_cv_version_from_env_with_host_registered():
+def test_positive_remove_cv_version_from_env_with_host_registered(
+    target_sat, default_org, rhel7_contenthost
```

Comment:
Most of the tests in this module use the `module_org` fixture. Is there a particular reason that you're using `default_org` in this test?